lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -7,17 +7,14 @@ define([
 ], function(Backbone, _, TopicTeamsView, TeamSpecHelpers, PageHelpers) {
     'use strict';
     describe('Topic Teams View', function() {
-        var createTopicTeamsView = function(options, isInstructorManagedTopic) {
-            var myTeamsCollection;
+        var createTopicTeamsView = function(options, isInstructorManagedTopic, showActions=true) {
             options = options || {}; // eslint-disable-line no-param-reassign
-            myTeamsCollection = options.myTeamsCollection || TeamSpecHelpers.createMockTeams({results: []});
             return new TopicTeamsView({
                 el: '.teams-container',
                 model: isInstructorManagedTopic ?
                     TeamSpecHelpers.createMockInstructorManagedTopic() : TeamSpecHelpers.createMockTopic(),
-                collection: options.teams || TeamSpecHelpers.createMockTeams(),
-                myTeamsCollection: myTeamsCollection,
-                showActions: true,
+                collection: options.teams || TeamSpecHelpers.createMockTeams({ results: [] }),
+                showActions: showActions,
```

Comment:
i cannot for the life of me figure out what the heck this variable is. It seems to only ever be set in test files?

Reply:
attempting to remove and see if the tests improve.  just a really long cycle time on the tests :-(